### ISSUE TITLE
Empty state icon not hidden from screen readers

### ISSUE THREAD

**Issue by metamoni** [joemasilotti/railsdevs.com#552]
## Describe the bug
The icons used in the `app/components/empty_state_component.html.erb` are decorative and should be hidden from screen readers. This component is used in several places, but you can test its behavior on the Notifications page.

## To reproduce

1. Clear your notifications if you have any
2. Go to the [notifications page](https://railsdevs.com/notifications)
3. Turn VoiceOver on: `command + F5`
4. Read out the page contents with `control + option + Right Arrow`
5. Notice that the bell icon is announced as `image` with no accessible title
6. Turn VoiceOver off: `command + F5`

## Expected behavior
VoiceOver should skip the bell icon

## Actual behavior
VoiceOver reads "image" without any additional context

## Screenshots
https://user-images.githubusercontent.com/22390758/178143470-8615bf51-5b77-4a86-be2f-3dc60b829e27.mov

## Additional context
- [Accessibility in inline_svg](https://github.com/jamesmartin/inline_svg#accessibility)
- [More on accessible SVGs](http://www.a11yonrails.com/images#accessible-svgs)

**Comment by crabbits** [comment]
@joemasilotti noticed this one is similar to #551 and #554 so happy to take it

**Comment by joemasilotti** [comment]
Closed by #562.
